src/migration/migrationRules.js

```diff
@@ -393,18 +365,10 @@ module.exports = {
           logger.info('Add swapWordsInDWords field to Modbus')
           dataSource.Modbus.swapWordsInDWords = false
         }
-      }
-    })
-  },
-  28: (config) => {
-    config.south.dataSources.forEach((dataSource) => {
-      if (dataSource.protocol === 'Modbus') {
-        dataSource.points.forEach((point) => {
-          if (!Object.prototype.hasOwnProperty.call(point, 'multiplierCoefficient')) {
-            logger.info('Add multiplierCoefficient field to Modbus points')
-            point.multiplierCoefficient = 1
-          }
-        })
+        if (!Object.prototype.hasOwnProperty.call(dataSource.Modbus, 'retryInterval')) {
+          logger.info('Add retryInterval field to Modbus')
+          dataSource.Modbus.swapWordsInDWords = 10000
```

Comment:
I think it should be dataSource.Modbus.retryInterval instead of dataSource.Modbus.swapWordsInDWord